File: php/Petshop.php
<?php
class PetShop {
    protected $id;
    protected $nama_produk;
    protected $harga_produk;
    protected $stok_produk;
    protected $foto_produk;

    public function __construct($id, $nama_produk, $harga_produk, $stok_produk, $foto_produk) {
        $this->id = $id;
        $this->nama_produk = $nama_produk;
        $this->harga_produk = $harga_produk;
        $this->stok_produk = $stok_produk;
        $this->foto_produk = $foto_produk;
    }

    public function getDetail() {
        return "
            <h3>{$this->nama_produk}</h3>
            <p>Harga: Rp {$this->harga_produk}</p>
            <p>Stok: {$this->stok_produk}</p>
            <img src='uploads/{$this->foto_produk}' width='150' />
            <hr>
        ";
    }
}
?>

File: Java/PetShop.java
// Parent class
public class PetShop {
    protected int id;
    protected String namaProduk;
    protected int hargaProduk;
    protected int stokProduk;

    public PetShop(int id, String namaProduk, int hargaProduk, int stokProduk) {
        this.id = id;
        this.namaProduk = namaProduk;
        this.hargaProduk = hargaProduk;
        this.stokProduk = stokProduk;
    }

    public void display() {
        System.out.printf("| %-5d | %-20s | Rp%-8d | %-5d |", id, namaProduk, hargaProduk, stokProduk);
    }
}
